### PR TITLE
Add framer-motion topic

### PR DESCRIPTION
**Pull request by rbourdon**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Framer motion is a popular animation library for react with almost 100k dependents and 1k public repositories using the topic.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by 07823226672 — COMMENTED** [review]
ᎪHMᎬD ᎪᏞHᎪRES

**Review by MikeMcQuaid — APPROVED** [review]

File: topics/framer-motion/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+aliases: motion, framer-motion
```
Comment (MikeMcQuaid) [inline topics/framer-motion/index.md:2]:
```suggestion
aliases: motion
```

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Thanks for the PR!

Also need to update the image: https://github.com/github/explore/runs/4392946945?check_suite_focus=true#step:7:418

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @rbourdon!

**Pull request merged**